src/components/StudentQuestion.js

```diff
@@ -14,15 +14,19 @@ const StudentQuestion = (props) => {
   const value = props.values[index];
   const error = props.errors[index];
   let { prompt } = props.questionData;
- 
 
   const handleOnChange = (e) => {
     props.setQuestionValue(index, e.target.value);
     props.resetQuestionError(index);
   }
 
-  const handleOnCheckboxChange = (val) => {
-    props.setQuestionValue(index, val);
+  const handleOnDropdownChange = (e, { value }) => {
```

Comment:
value should not be in curly brackets - breaks page because it thinks you are trying to destructure

Reply:
Here I just changed it back to how it was in your PR